pkgs/by-name/sp/spotube/package.nix

```diff
@@ -111,8 +116,19 @@ let
     postFixup = ''
       makeWrapper $out/share/spotube/spotube $out/bin/spotube \
           "''${gappsWrapperArgs[@]}" \
-          --prefix LD_LIBRARY_PATH : $out/share/spotube/lib:${lib.makeLibraryPath [ mpv-unwrapped ]} \
-          --prefix PATH : ${lib.makeBinPath [ xdg-user-dirs ]}
+          --prefix LD_LIBRARY_PATH : $out/share/spotube/lib:${
+            lib.makeLibraryPath [
+              glib
+              gtk3
+              libGL
+              libsoup_3
+              mesa
+              mpv-unwrapped
+              webkitgtk_4_1
+            ]
+          } \
+          --prefix PATH : ${lib.makeBinPath [ xdg-user-dirs ]} \
+          --set LIBGL_ALWAYS_SOFTWARE 1
```

Comment:
For me, only `mpv-unwrapped` is needed inside `makeLibraryPath`'s list.

Also, `LIBGL_ALWAYS_SOFTWARE` doesn't seem to be needed for me...
Is this perhaps some X11/wayland difference?